staging/src/k8s.io/apiserver/pkg/admission/attributes.go

```diff
@@ -44,7 +43,7 @@ type attributesRecord struct {
 
 	// other elements are always accessed in single goroutine.
 	// But ValidatingAdmissionWebhook add annotations concurrently.
-	annotations     map[string]audit.Annotation
+	annotations     map[string]Annotation
```

Comment:
why do we need this as a public concept? It's fine here as a private type. But I would not use it in any func signature.